ministryofjustice/correspondence_tool_staff
# CT-3239 Added pretty_type: offender SAR and complaint case type

## Description

This PR is to make the type column on open/closed cases tab look more straightforward and nice

## Self-review checklist


### Screenshots


### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3239
### Deployment


### Manual testing instructions

1 - Login in as Branston user
2 - check the 'type' columns for offender sar case and complaint case
